From 22bdc91da072a0f460cfba8b5bf30d387591b00e Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Tue, 6 Feb 2018 22:44:25 -0500
Subject: [PATCH 1248/1936] Update rtosc/instruments submodules

- Adds log scale parameters to automations
- Updates instruments and fixes a long standing spelling error in
  the arpeggios
---
 instruments | 2 +-
 rtosc       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/instruments b/instruments
index 2f5a5792..22ea12d5 160000
--- a/instruments
+++ b/instruments
@@ -1 +1 @@
-Subproject commit 2f5a57925cfad1d99a7ca6b2e96e61b55db18358
+Subproject commit 22ea12d55e985eee6b57fcac1cafb7df6ae392f2
diff --git a/rtosc b/rtosc
index 36b5301d..60399ae0 160000
--- a/rtosc
+++ b/rtosc
@@ -1 +1 @@
-Subproject commit 36b5301d7c7112a1677159c13b751d647eb2ced1
+Subproject commit 60399ae0d95c53d2e03931afc10cfd6949e403ba
-- 
2.47.0

